packages/blueprints-integration/src/content.ts

```diff
@@ -9,6 +9,8 @@ export type WithTimeline<T extends BaseContent> = T & {
 export interface BaseContent {
 	editable?: BaseEditableParameters
 
+	loop?: boolean
```

Comment:
Could you add a description for this property? Like what's the intended use, perhaps an example ("looping graphics background"?) and describe what this changes (functionality, GUI etc?).